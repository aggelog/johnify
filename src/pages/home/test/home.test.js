import React from 'react';
import Home from '../home';
import renderer from 'react-test-renderer';

jest.mock('../../../containers/searchBar', () => 'SearchBar');
jest.mock('../../../containers/gifsGrid', () => 'GifsGrid');

it('renders correctly', () => {
  const tree = renderer
    .create(<Home globalLoading={false} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});