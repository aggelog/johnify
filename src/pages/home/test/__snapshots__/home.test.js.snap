// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-12 col-md-12 col-lg-12"
      >
        <SearchBar />
      </div>
    </div>
    <GifsGrid />
  </div>
</div>
`;
